tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist =
+    py{310, 39, 38, 37}
```

Comment:
Should 3.11(-dev) be included here too?

Reply:
Let's add 3.11 in another PR.

I've got it almost working for 3.11, but there's a doctest failure I'm not sure about so I can open a draft PR after this is merged.